tests/unit/test-util/test.adb.js

```diff
@@ -622,6 +632,35 @@ describe('utils/adb', () => {
        });
   });
 
+
+  describe('checkOrCleanArtifacts', () => {
+    it('cleans the previously artifacts directory when condition true' +
+      ' but not removed earlier', async () => {
```

Comment:
splitting the description into two strings and composing them with + looks a bit bad, I think we can just make the description shorter (e.g. "it(`removes old artifacts directories`, ...)")

also, we need to test the helper function on at least two scenarios: when it does only check if any old artifacts exists and when it does also remove them.